mm2src/mm2_main/tests/mm2_tests/lightning_tests.rs

```diff
@@ -61,6 +61,7 @@ fn start_lightning_nodes(enable_0_confs: bool) -> (MarketMakerIt, MarketMakerIt,
             "estimate_fee_mode": "ECONOMICAL",
             "mm2": 1,
             "required_confirmations": 0,
+            "avg_blocktime": 600,
```

Comment:
Could you use constant variable for all `avg_blocktime` fields in this module for same reason as https://github.com/KomodoPlatform/atomicDEX-API/pull/1601#discussion_r1066921814

Reply:
Done